app/assets/javascripts/_selection-buttons.js

```diff
@@ -0,0 +1,16 @@
+(function(GOVUK, GDM) {
+
+  GDM.selectionButtons = function() {
```

Comment:
Not for this PR necessarily, but would this kind of script sit happily in the frontend toolkit?